components/brave_ads/core/internal/settings/settings.cc

```diff
@@ -44,6 +45,7 @@ int GetMaximumNotificationAdsPerHour() {
   int ads_per_hour =
       static_cast<int>(AdsClientHelper::GetInstance()->GetInt64Pref(
           prefs::kMaximumNotificationAdsPerHour));
+  CHECK_NE(0, ads_per_hour);
```

Comment:
If this crash occurs, the user might not have a way to get out of it without manually editing prefs. It might be better just to return the default ads per hour here (or maybe for any value less than or equal to zero), and rely on the user to change the value to what they want.

Or, you could just change it here to something valid.

Reply:
@zenparsing I have made a change that any value <= 0 returns the default. Thanks